### PR TITLE
cover for blog

### PR DESCRIPTION
**Pull request by ngoc-nguyenminh**

### CODE REVIEW

File: resources/js/cover.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const dummyCover = 'resources/images/dummyimage1.jpg';
+const str = '#BLOG';
+
+let coverBackground = document.getElementById('cover');
```
Comment (NakSin) [inline resources/js/cover.js]: why using let ? :v

Comment (ngoc-nguyenminh) [reply resources/js/cover.js]: Because of Duy's advices and as far as i khow, 'var' is function scoped while 'let' is block scoped

Comment (MiloNguyen95) [reply resources/js/cover.js]: We should use const here. Additionally, coverBlock is a better name instead of coverBackground

Comment (MiloNguyen95) [reply resources/js/cover.js]: @ngoc-nguyenminh fix này để a còn merge e

Comment (ngoc-nguyenminh) [reply resources/js/cover.js]: @MiloNguyen95  dạ e fix rồi đó a, sorry anh e tối qua ngủ sớnm quá :(((

**Pull request merged**